apps/server/src/controllers/riderController.ts

```diff
@@ -0,0 +1,55 @@
+import { Request, Response } from 'express';
+import Ryder from "../models/ryder";
+
+// Controller function for registering riders
+export const registerRider = async (req: Request, res: Response): Promise<void> => {
+    try {
+        const { name, email, phone, vehicleType } = req.body;
+
+        if (!name || !email || !phone || !vehicleType) {
+            res.status(400).send('Name, email, phone, and vehicleType are required');
+            return;
+        }
+
+        // You can perform additional validation or saving to the database here if needed
+
+        res.status(201).json({ message: 'Rider registered successfully' });
+    } catch (error) {
+        console.error('Error registering rider:', error);
+        res.status(500).send('Internal server error');
+    }
+};
```

Comment:
The same thing applies to this. Remove this function entirely. There's an existing function for that.